libraries/AP_Scripting/lua_scripts.cpp

```diff
@@ -504,10 +504,12 @@ void lua_scripts::run(void) {
     // Skip those directores disabled with SCR_DIR_DISABLE param
     uint16_t dir_disable = AP_Scripting::get_singleton()->get_disabled_dir();
     bool loaded = false;
+#if AP_FILESYSTEM_FILE_WRITING_ENABLED
```

Comment:
Shouldn't this use `AP_FILESYSTEM_FILE_READING_ENABLED`?

Reply:
Fixed